resolwe_bio/processes/reads_processing/trimmomatic.yml

```diff
@@ -0,0 +1,415 @@
+---
+
+- slug: trimmomatic-single
+  name: trimmomatic (single-end)
+  data_name: '{{ reads.fastq.0.file|default:"?" }}'
+  version: 0.0.1
+  type: data:reads:fastq:single:trimmomatic
+  category: analyses
+  flow_collection: sample
```

Comment:
OK, thank you both for the clarification. @JenkoB please revert this change back to the way it was before.